### PR TITLE
Feat/junit5 hibernate

### PR DESCRIPTION
**Pull request by christopher-cudennec**

### CODE REVIEW

File: sda-commons-server-testing/README.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ In case you want to use JUnit 5 you also have to activate it in your build.gradl
   test {
     useJUnitPlatform()
   }
+
+  // in case you use the integrationTest plugin:
+  integrationTest {
+    useJUnitPlatform()
```
Comment (ericrath) [inline sda-commons-server-testing/README.md:22]: we might want to use inheritance here
let me check the syntax

**Review by ericrath — CHANGES_REQUESTED** [review]

File: sda-commons-server-testing/README.md (hunk 1 above)
Comment (ericrath) [reply sda-commons-server-testing/README.md:22]: configurations {
    integrationTest.extendsFrom test   
}

**Review by ericrath — CHANGES_REQUESTED** [review]

File: sda-commons-server-testing/README.md (hunk 1 above)
Comment (RappC) [reply sda-commons-server-testing/README.md:22]: @ericrath what is the advantage of your proposal over the suggestion made by @christopher-cudennec

Comment (ericrath) [reply sda-commons-server-testing/README.md:22]: it makes sure, that our integration tests dont differ from the junit base.
Especially if we add functionality to test{}

Comment (christopher-cudennec) [reply sda-commons-server-testing/README.md:22]: I think I would go with your suggestion if there was more configuration to the test plugin. 
For now it's the only thing that needs to be configured, so I would stick with the current version.

**Review by ericrath — APPROVED** [review]

**Review by JoergSiebahn — APPROVED** [review]

**Pull request merged**